RUNBOOK.md

```diff
@@ -0,0 +1,153 @@
+# manage-recalls-ui Run Book
+
+## Service or system overview
+
+### Business overview
+
+This provides users with a service for creating and managing recalls within HMPPS. (This is the UI for the companion application [manage-recalls-ui](https://github.com/ministryofjustice/manage-recalls-ui)).
+
+### Technical overview
+
+Internal application based on Express.js, using Typescript (compiled to javascript running on node.js) as the main language. Deployed in kubernetes (the HMPPS Cloud Platform) using the configuration found in [helm_deploy](helm_deploy).
+
+### Service Level Agreements (SLAs)
+
+Office hours (Mon-Fri, 09:00-17:00), best efforts.
+
+### Service owner
+
+The `ppud-replacement` team develops and runs this service.
+
+Contact the [#ppud-replacement](https://mojdt.slack.com/archives/C020S8C0K9U) and [#ppud-replacement-devs](https://mojdt.slack.com/archives/C0223AGGQU8) channels on slack.
+
+### Contributing applications, daemons, services, middleware
+
+- Express.js application based on [hmpps-template-typescript](https://github.com/ministryofjustice/hmpps-template-typescript).
+- AWS Elasticache (redis) for caching - configured via [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments).
+
+## System characteristics
+
+### Hours of operation
+
+Available 24/7.
+
+### Infrastructure design
+
+The application runs on the [HMPPS Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) within the `manage-recalls-<tier>` namespaces (where `tier` is `dev`, `preprod` or `prod`).
+
+**NOTE:** The service runs on the `live` cluster, not the deprecated `live-1` instance.
+
+The main application runs as a deployment named `manage-recalls-ui`. The application is made avialable externally from the cluster via an Ingress.
+
+See the `values-<tier>.yaml` files in the [helm_deploy](helm_deploy) directory for the current configuration of each tier.
+
+### Throttling and partial shutdown
+
+If there is an issue with the service where it is causing load on downstream services and it needs to be shutdown quickly the following command will reduce the number of pod replicas to zero:
+
+```
+kubectl -n manage-recalls-<tier> scale deployment manage-recalls-ui --replicas=0
+```
+
+We do not currently have a strategy in place to throttle requests.
+
+### Environmental differences
+
+Infrastructure wise, all three tiers are identical, but `prod` has the following differences:
+
+- It will have more pod replicas of the main application deployment.
+- As this is live data, you **must** be SC cleared to work on it or access the data held within it.
```

Comment:
Could someone take from this that they need to be SC-cleared to work on the app ie make a change and deploy it to prod?

Reply:
Hmm, possibly...  Yeah, let me word it better